spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -40,41 +41,58 @@
 import org.springframework.beans.factory.InitializingBean;
 import org.springframework.cloud.gcp.pubsub.support.PublisherFactory;
 import org.springframework.cloud.gcp.pubsub.support.SubscriberFactory;
+import org.springframework.cloud.gcp.pubsub.support.converter.JacksonPubSubMessageConverter;
+import org.springframework.cloud.gcp.pubsub.support.converter.PubSubMessageConverter;
 import org.springframework.util.Assert;
 import org.springframework.util.concurrent.ListenableFuture;
 import org.springframework.util.concurrent.SettableListenableFuture;
 
 /**
+ * // @formatter:off
```

Comment:
Why do we need this?

Seems for me I fully don't use formatting for JavaDocs in my IDEA - to much inconsistency with required code style.
I prefer to reformat it manually.

Reply:
My auto-formatter (using the xml style file in this repo) will move that `<p>` to another line in certain situations (including this one), which fails the checkstyle.

Reply:
Well, then we have to fix that auto-formatter, because having `<p>` on the same like as fist sentence of the paragraph is indeed our code style: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting.

Let's just try do not use this IDE-specific `@formatter:off`!